VoteSystem.Console/Program.cs

```diff
@@ -100,7 +100,7 @@ static void Main(string[] args)
                             #endregion
                             #region AddChoice
                             case 2:
-                                ChoiceCreationDTO choiceCreation = userInterface.CreateChoiceConsole(user_temp_id);
+                                ChoiceCreationDTO choiceCreation = userInterface.CreateChoiceConsole();
```

Comment:
there is a lot of places like this where you'll be able to remove a method argument because it reflects the "current user", which can be obtained from registrationContext (although, you'd better rename it to "authorizationContext")